src/design/optForce/analizeOptForceSol.m

```diff
@@ -0,0 +1,131 @@
+function [maxGrowthRate, minTarget, maxTarget] = analizeOptForceSol(model, targetRxn, solution, relax, tol)
```

Comment:
Yes, generally you will have to run `contribute` [3] for a simple file move and then only do modifications in the file and re-run `contribute` [3] to publish the changes. This is standard `git` behaviour though. 

You would have 2 commits (1 move and 1 change commit) then, which is easier to track for reviewers.